translations/translations.json

```diff
@@ -11,7 +11,8 @@
       "createPdf": "PDF ይፍጠሩ",
       "metaDescription": "{{appName}} በጀርመን ላለው ህይወት የእርስዎ ዲጂታል መመሪያ ነው። አካባቢያዊ መረጃን፣ ክንዉኖችን እና ምክርን ያግኙ። በእርስዎ ቋንቋ ሁልጊዜ የቅርብ ጊዜ የሆነ እና የሚገኝ ነው።",
       "organizationContent": "የዚህ ገጽ ይዘቶች ከ{{organization}} ናቸው።",
-      "organizationMoreInformation": "ስለ <1>{{organization}}</1> on <3>{{domain}}</3> ተጨማሪ መረጃ ማግኘት ይችላሉ።"
+      "organizationMoreInformation": "ስለ <1>{{organization}}</1> on <3>{{domain}}</3> ተጨማሪ መረጃ ማግኘት ይችላሉ።",
+      "disabledPdf": "/ፒዲኤፍ/pdf ወደ ውጭ መላክ በአሁኑ ጊዜ በዚህ ቋንቋ አይቻልም።"
```

Comment:
@osmers a few translations in tygrinia seem to be off, for example this one. Could you perhaps double check?

Reply:
@steffenkleinle could you define - off? What would you reckon is wrong?

Reply:
If you paste this to google translate, you get:
> /pdf/pdf export is currently not possible in this language.

I had some other example where I thought something was wrong but can't find it anymore :see_no_evil:
And obviously I meant Amharic, not Tygrinia.

Reply:
@steffenkleinle bei Senden und Du wird mir auch was anderes übersetzt...kann natürlich an Google Translate liegen, aber ich frage mal nach. 
Ist dir noch bei anderen Sprachen was aufgefallen?

Reply:
Ne, ich glaub nur hier.

Reply:
@steffenkleinle eine Sache ist mir noch eingefallen - alle Übersetzungen in Amharisch sind mit Sie übersetzt, während wir im Deutsch Du schreiben. Stört uns das? Scheint aber auch in den alten Übersetzungen schon so zu sein.

Reply:
Hmm... das ist eher ne Frage für z.B. @dkehne denke ich.

Reply:
Ist auch die Frage, ob es sowas wie Du überhaupt gibt bzw kulturell angebracht ist. Ich würde es jetzt erstmal so lassen, weil wir sonst alle überarbeiten müssten, wollte aber mal drauf hinweisen. 

Die anderen Punkte klärt MeinRad bereits mit dem Übersetzer :)